packages/api-nest/src/library/library.service.ts

```
@@ -705,6 +705,32 @@ export class LibraryService {
 
+  /**
+   * Update notebook content for a library item
+   * @param userId - User ID who owns the item
+   * @param itemId - Library item ID
+   * @param note - Notebook content (supports markdown)
+   * @returns Updated library item
+   */
+  async updateNotebook(
+    userId: string,
+    itemId: string,
+    note: string,
```

ESLint

[error] Delete `,`

(prettier/prettier)